libass/ass.c

```diff
@@ -71,12 +71,6 @@ void ass_free_track(ASS_Track *track)
     if (!track)
         return;
 
-    if (track->parser_priv) {
-        free(track->parser_priv->read_order_bitmap);
-        free(track->parser_priv->fontname);
-        free(track->parser_priv->fontdata);
-        free(track->parser_priv);
-    }
```

Comment:
Since `events_size` no longer exists there’s no more reason to move this code, right?